README.md

```diff
@@ -291,6 +284,71 @@ One other tip... This totally works:
 </glamorous.Div>
 ```
 
+#### Overriding component styles
+
+The most common scenario for using props is to override the style of an
+existing component (generated by `glamorous` or not). In this section we'll
+explain how you can use `className`, `css` and composition to override the
+styles of a component. Let's see how that can be done in the examples below.
+
+> Try this out in your browser [here](https://codesandbox.io/s/Kro5369wG)!
```

Comment:
Oh, one other thing on your codesandbox, make sure to change the name and the description so it's not "glamorous help" but it's "glamorous example" or something more descriptive :)